Week 2 Reflection- Carley Gilmore

For this week's reflection, I surfed Twitter under the hashtag #dataviz to find a post by Dr. Jean Fan, an assistant professor at John Hopkins on spatial transcriptomics. The blog post can be found at this link: https://jef.works/blog/2022/01/19/exploring-umap-parameters/

The blog shows how different UMAP parameters can change the way single cell spatial transcriptomics are  expressed. I googled about spatial transcriptomics and learned that it is a term for a range of methods designed for assigning cell types to their appropriate locations. It is a molecular profiling method that allows scientists to measure gene activity and map where the activity is occurring. This kind of research is monumental in aiding in our understanding of diseases and other biological processes. What interested me the most was that by manipulating the spread and minimum distance of the plots with different UMAP parameters, the final 2D visualization changes. The breakthrough here is that different combinations of those parameters can show different impessions of how different the clusters of cells are. I also think that the animation of the embeddings for showing how the cells transition is particularly valuable. Visualizing this data in different ways can help detect which genes drive certain behaviors.